From: Brian May <bam@debian.org>
Date: Sun, 6 Nov 2016 09:51:19 +1100
Subject: Update reference to python-werzeug package

Point people towards installing the python-werzeug package, not to
download random files from the internet.

Forwarded: not-needed
---
 django_extensions/management/commands/runserver_plus.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/django_extensions/management/commands/runserver_plus.py b/django_extensions/management/commands/runserver_plus.py
index 3f69485..6181f97 100644
--- a/django_extensions/management/commands/runserver_plus.py
+++ b/django_extensions/management/commands/runserver_plus.py
@@ -305,7 +305,7 @@ class Command(BaseCommand):
 
     def inner_run(self, options):
         if not HAS_WERKZEUG:
-            raise CommandError("Werkzeug is required to use runserver_plus.  Please visit http://werkzeug.pocoo.org/ or install via pip. (pip install Werkzeug)")
+            raise CommandError("Werkzeug is required to use runserver_plus.  Please install `python3-werkzeug'")
 
         # Set colored output
         if settings.DEBUG:
